### PR TITLE
Do not import `pg` handler explicitly

### PR DESCRIPTION
**Pull request by AlekSi**
To avoid conflicts between #784 and #796.

## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [x] I ran `task all`, and it passed.

### CODE REVIEW

**Review by rumyantseva — DISMISSED** [review]
Yeah, makes sense!

File: integration/go.mod [hunk 1]
```diff
@@ -13,14 +13,26 @@ require (
 )
 
 require (
```
Comment (AlekSi) [inline integration/go.mod:15]: Deps were messed up by the previous commit

**Comment by AlekSi** [comment]
@rumyantseva PTAL

File: integration/go.mod (hunk 1 above)
Comment (rumyantseva) [reply integration/go.mod:15]: Do you understand where are all the new indirect dependencies from?

Comment (AlekSi) [reply integration/go.mod:15]: From the Tigris handler. That's not a problem in integration tests

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**